### ISSUE TITLE
Feature Request: Baseurl support for panel itself

### ISSUE THREAD

**Issue by flying-sausages** [liaralabs/swizzin_dashboard#25]
It's actually not a terrible idea to support something like `domain.tld/panel` or `domain.tld/login` if people don't want the panel to be on the domain's root. This would allow some security by obscurity. I would be interested in this myself. I would not provide any installer or nginx support for this beyond docs.

**Comment by wii747** [comment]
+1 would be a good idea

**Comment by fablb** [comment]
+1

**Comment by liaralabs** [comment]
Implemented for testing in #35 

New config option: `URLBASE`

Valid Usage:
```
URLBASE = '/panel`
URLBASE = '/'
URLBASE = ''
URLBASE = '/foo/bar'
```

**Comment by flying-sausages** [comment]
Added in #35
